src/hashtables.jl

```diff
@@ -26,11 +27,13 @@ end
 function show_params_TaylorN()
     info( """`TaylorN` and `HomogeneousPolynomial` parameters:
     Maximum order       = $(_params_taylorN.maxOrder)
-    Number of variables = $(_params_taylorN.numVars)""" )
+    Number of variables = $(_params_taylorN.numVars)
+    Variable names = $(_params_taylorN.variable_names)
+    """)
     nothing
 end
 
-global const _params_taylorN = ParamsTaylorN(6,2)
+global const _params_taylorN = ParamsTaylorN(6,2, ["x", "y", "z", "t"])
```

Comment:
`t` is used in the display of `Taylor1` series, so the last entry should be something different, "w", for instance.

Two more related question: How can one change those names? And, isn't it odd that we  we have two independent variables, but we are defining four?